riggerlib/client.py

```diff
@@ -115,6 +115,7 @@ def terminate(self):
         try:
             self._request({'event_name': 'shutdown'})
             self.ready = False
+            self.zmq.close()
```

Comment:
this closes the socket on only one thread, other threads will be unaffected
i do wonder why this works